code-of-conduct/enforcement-manual.md

```diff
@@ -0,0 +1,66 @@
+# 4 Enforcement Manual
+
+This is the enforcement manual followed by OBF.
+It's used when we respond to an issue to make sure we're consistent and fair.
+Enforcement of the Code of Conduct should be respectful and not include any harassing behaviors.
+
+## 4.1 Urgent Situations: Acting Unilaterally
+
+If the incident involves physical danger or involves a threat to anyone's safety (e.g. threats of violence), any member of the community may -- and should -- act unilaterally to protect the safety of any community member.
+This can include contacting law enforcement (or other local personnel) and speaking on behalf of the OBF.
+
+If the act is ongoing, any community member may act immediately, before reaching consensus, to diffuse the situation.
+In ongoing situations, any member may at their discretion employ any of the tools available in this enforcement manual, including bans and blocks online, or removal from a physical space.
+
+In situations where an individual community member acts unilaterally, they must inform the appointed members as soon as possible, and report their actions for review within 24 hours.
+
+## 4.2 Less-Urgent Situations
+
+Upon receiving a report of an incident, the Code of Conduct committee will review the incident and determine, to the best of their ability:
+
+- whether this is an ongoing situation
+- whether there is a threat to anyone's physical safety
+- what happened
+- whether this event constitutes a Code of Conduct violation
+- who, if anyone, was the bad actor
+
+This information will be collected either in person or in writing.
+The Code of Conduct committee will provide a written summary of the information surrounding the incident.
+All participants will be anonymized in the summary report, referred to as "Community Member 1", "Member project 1", or "Board member 1" and so on.
+The "de-anonymizing key" will be kept in a separate file and only accessed to link repeated reports against the same person over time.
+
+The Code of Conduct committee will aim to have a resolution agreed upon within one week.
+If a resolution can't be determined in that time, a member of the Code of Conduct committee will respond to the reporter(s) with an updated and projected timeline for resolution.
+
+## 4.3 Resolutions
+
+The Code of Conduct committee will seek to agree on a resolution by consensus of all members investigating the report in question.
+If the committee cannot reach consensus and deadlocks for over a week, the appointed members from the OBF board will break the tie.
```

Comment:
```suggestion
If the committee cannot reach consensus and deadlocks for over a week, the appointed members from the OBF Board will break the tie.
```

Reply:
Having _consensus_, _deadlock_, and _tie_ appear in the same sentence in this way is confusing. Does _deadlock_ mean one of the committee members refuses to join what is the consensus of the other committee members (in which case there is no tie that needs to be broken), or does deadlock mean there is a tie that needs to be broken (in which case there is no consensus, and tie-breaking won't produce a consensus).